QuadradiusR/scripts/torus.gd

```diff
@@ -85,18 +124,27 @@ func _end_drag():
 	rect_position = self.starting_pos
 
 
+# # # # #
+# Utils #
+# # # # #
 func set_interaction(can_interact):
 	if not is_held:
 		self.can_interact = can_interact
-		collision_detector.disabled = !can_interact
+#		collision_detector.disabled = !can_interact
```

Comment:
this is disabled so that you can see your powerups even if its not your turn, should probably be removed altogether